packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/options/components/activation-status/activation-status.html

```diff
@@ -15,7 +15,7 @@
     </div>
     <div
         class="d-inline-block"
-        data-ng-repeat="datacenter in ::$ctrl.datacentersState track by index"
+        data-ng-repeat="datacenter in ::$ctrl.datacentersState"
```

Comment:
**question:** Any reason why this has been removed? Imo, we can keep `track by` which will have better performance.

Reply:
In fact it isn't linked with tracking, i saw this issue, in browser console i had message unique key is undefined and the information in tile wasn't display.

Reply:
I fixed it. In fact 
data-ng-repeat="datacenter in ::$ctrl.datacentersState track by index"

data-ng-repeat="datacenter in ::$ctrl.datacentersState track by $index"

there was $ that missed